MekHQ/resources/mekhq/resources/PartsInUseTableModel.properties

```diff
@@ -10,3 +10,4 @@ ordered.heading=Ordered
 part.heading=Part
 stored.heading=Stored
 storedTonnage.heading=Tonnage
+requestedStock.heading =Requested Stock Percent
```

Comment:
Extra whitespace after `.heading`

Reply:
In fact, I've noticed a lot of weird whitespacing throughout your code. Lots of `if(` or `while(` instead of `if (` and `while (` which is more standard for our codebase. I'm not going to force you to change them, but I do think we need to maintain whitespace consistency where possible.